samcli/commands/local/lib/local_lambda.py

```diff
@@ -206,7 +220,7 @@ def _make_env_vars(self, function):
         name = function.name
 
         variables = None
-        if function.environment and isinstance(function.environment, dict) and "Variables" in function.environment:
```

Comment:
Why is function now guaranteed non-`None`?

Reply:
resolved after discussing offline